contrib/init/openrc/docker.initd

```diff
@@ -2,9 +2,9 @@
 # Copyright 1999-2013 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-command="${DOCKER_BINARY:-/usr/bin/docker}"
+command="${DOCKER_BINARY:-/usr/bin/dockerd}"
```

Comment:
I wasn't sure what we ought to do here -- if folks have an existing `/etc/conf.d/docker`, then this means that their `DOCKER_BINARY` entry now needs to change to point to `dockerd` instead, so I think we should probably rename the var to make this really explicit and clear (and update `docker.confd` appropriately).